modules/elasticsearch/versions.tf

```
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">= 0.13.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.8"
+    }
+    template = {
+      source  = "hashicorp/template"
+      version = ">= 2.0"
+    }
+    null = {
+      source  = "hashicorp/null"
+      version = ">= 2.0"
+    }
+  }
```

```suggestion
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = ">= 3.8"
    }
  }
```

I didn't run `tflint` on this component, but I'm fairly sure only the `aws` provider is used.